### ISSUE TITLE
Training,loss Bigger，traing my custom dataset；

### ISSUE THREAD

**Issue by bsysman** [AlexeyAB/darknet#7982]
train.cfg
[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=8
subdivisions=8
width=896
height=896
channels=3
momentum=0.949
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 500500
policy=steps
steps=400000,450000
scales=.1,.1

**train info**
[yolo] params: iou loss: ciou (4), iou_norm: 0.05, obj_norm: 1.00, cls_norm: 0.50, delta_norm: 1.00, scale_x_y: 2.00
Total BFLOPS 324.651 
avg_outputs = 1684006 
 Allocate additional workspace_size = 86.34 MB 
Loading weights from data/cfg/yolov4-p5.conv.232...0,1
37
net.optimized_memory = 0 
mini_batch = 1, batch = 1, time_steps = 1, train = 1 
Create CUDA-stream - 0 
 Create cudnn-handle 0 
nms_kind: diounms (2), beta = 0.600000 
nms_kind: diounms (2), beta = 0.600000 
nms_kind: diounms (2), beta = 0.600000 
Done! Loaded 233 layers from weights-file 
 1 : compute_capability = 750, cudnn_half = 1, GPU: GeForce RTX 2080 Ti 

![chart](https://user-images.githubusercontent.com/25891803/128618906-af46653e-cbcb-4437-bd93-d00eee3b1164.png)

**Comment by mey1k** [comment]
it looks like overfitting..

I think train dataset is wrong.
you have to re-labeling, and calc_anchors.

Recommended iteration count = class count * 2000

**Comment by bsysman** [comment]
The same dataset,yoloV4 train is ok,I tried many times, all with the same structure.
set count = class count * 2000 and batch=64.
